torch/testing/_internal/dynamo_test_failures.py

```diff
@@ -3178,6 +3172,123 @@
+    "TestProfiler.test_record_function_fast",  # profiler/test_profiler
+    "TestProfiler.test_source_multithreaded_close_in_scope_work_in_main_thread_False",  # profiler/test_profiler
+    "TestProfiler.test_profiler_fwd_bwd_link",  # profiler/test_profiler
+    "TestProfiler.test_concrete_inputs_profiling",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_tensorimpl_invalidation_scalar_args",  # profiler/test_profiler
+    "TestProfiler.test_guarded_record_function_fast",  # profiler/test_profiler
+    "TestExperimentalUtils.test_profiler_optimizer_single_tensor_pattern",  # profiler/test_profiler
+    "TestExperimentalUtils.test_utils_compute_self_time",  # profiler/test_profiler
+    "TestProfiler.test_high_level_trace",  # profiler/test_profiler
+    "TestRecordFunction.test_datapipe_with_record_function_fork",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_allocations",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_module_and_optimizer_ids",  # profiler/test_profiler
+    "TestExperimentalUtils.test_utils_compute_queue_depth_when_no_cuda_events",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_allocation_id_uniqueness",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_sparse_tensors",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_optimizer",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_tensorimpl_invalidation_keep_alive",  # profiler/test_profiler
+    "TestExperimentalUtils.test_utils_compute_queue_depth",  # profiler/test_profiler
+    "TestExperimentalUtils.test_profiler_pattern_match_helper",  # profiler/test_profiler
+    "TestProfiler.test_export_stacks",  # profiler/test_profiler
+    "TestProfiler.test_source_multithreaded_basic_work_in_main_thread_True",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_mkldnn_tensors",  # profiler/test_profiler
+    "TestRecordFunction.test_datapipe_with_record_function",  # profiler/test_profiler
+    "TestProfiler.test_memory_profiler",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_tensor_lists",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_pointers_and_ids",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_nnmodule_params",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_tensorimpl_invalidation_set",  # profiler/test_profiler
+    "TestProfiler.test_source_multithreaded_basic_work_in_main_thread_False",  # profiler/test_profiler
+    "TestTorchTidyProfiler.test_scalar_ins",  # profiler/test_profiler
+    "TestProfiler.test_profiler_op_event_args",  # profiler/test_profiler
+    "TestProfiler.test_source_multithreaded_complex_work_in_main_thread_True",  # profiler/test_profiler
+    "TestProfiler.test_source_multithreaded_multiple_preexisting_work_in_main_thread_True",  # profiler/test_profiler
+    "TestAOTAutograd.test_input_mutation_aliases_and_output_alias",  # functorch/test_aotdispatch
```

Comment:
Hmm does this mean that dynamo is running directly on the `test_aotdispatch.py` tests? Do we want that? (having dynamo run on the AOTAutograd code seems.. fragile)

Reply:
Answered offline, but Dynamo should graph break correctly (or handle them) when it sees things like make_fx, FunctionalTensor, FakeTensor, etc.